packages/tailwindcss/src/index.ts

```diff
@@ -0,0 +1,10 @@
+import plugin from 'tailwindcss/plugin'
+
+const formKitStates = plugin(({ addVariant }) => {
+  addVariant('formkit-invalid', ['[data-type][data-invalid]', '[data-type][data-invalid] &'])
+  addVariant('formkit-errors', ['[data-type][data-errors]', '[data-type][data-errors] &'])
+  addVariant('formkit-disabled', ['[data-type][data-disabled]', '[data-type][data-disabled] &'])
+  addVariant('formkit-complete', ['[data-type][data-complete]', '[data-type][data-complete] &'])
```

Comment:
Possible way to do the above???
```suggestion
  addVariant('formkit-complete', ['[data-type][data-complete]', '[data-type][data-complete] &'])
  addVariant('formkit-complete', ['[data-type="button"],[data-type="submit"][data-loading]', '[data-type="button"],[data-type="submit"][data-loading] &'])
```

Reply:
That would solve it for the button, but users may want to do other treatments (disable pointer events, drop opacity, etc) across the entire form. Ideally we'd add it as a prop at the top level like the others so that it can be used anywhere and users don't have to think about which variants are available where.